include/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp

```diff
@@ -256,6 +237,7 @@ struct buffered_piece_collection
 
     struct original_ring
     {
+        typedef geometry::model::box<robust_point_type> robust_box_type;
```

Comment:
In the next (smaller) step I will update point-in-original-ring with non rescaled types